groups/groups.yaml

```diff
@@ -523,8 +523,8 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
-    - jichenjc@cn.ibm.com
-    - sbueringer@gmail.com
+      - jichenjc@cn.ibm.com
```

Comment:
Can you please avoid these white space changes?

Reply:
Sorry about. Done